### PR TITLE
Tag v0.11.0

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by HackFisher** [comment]
Need to check HeaderMMR pallet storage name, if dependency is using new  frame v2 

https://github.com/darwinia-network/darwinia/blob/711c8c5c38b9288ab3aa99a297a36bbb5b190294/runtime/crab/src/lib.rs#L192 

Might be `"HeaderMMR" --> "DarwiniaHeaderMMR"`

**Review by HackFisher — CHANGES_REQUESTED** [review]
Due to the changes in https://github.com/darwinia-network/darwinia-common/pull/673 and

https://github.com/darwinia-network/darwinia-common/compare/darwinia-v0.10.0..xavier-darwinia-v0.10.1

This darwinia-common companion and maybe also migration need to be upgraded.

Need carefully review considering the changes in darwinia-v0.10.1

**Review by HackFisher — CHANGES_REQUESTED** [review]
Current base branch `master` is using the deprecate commit https://github.com/darwinia-network/darwinia-common/commit/4e2a5dff2b30cb37a404466f00953767a75b1980 before mmr patch.

After the runtimes in v0.10.1 be applied, the dependency will require to be changed to commit after mmr patch(using new header mmr pallet).

https://github.com/darwinia-network/substrate-update-tracking/issues/19

If we use commits after darwinia-v0.11.1, check the migration differences between these substrate version(companion polkadot can be used as reference).

**Comment by HackFisher** [comment]
Related polkadot release with same substrate dependency:

https://github.com/paritytech/polkadot/releases/tag/v0.9.0

What's the changes cause the mandatary upgrade for the nodes?

The new host function mentioned in polkaodt v0.9.0? Any related pull request links to this break changes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+### Change Log
+
+#### Update(s)
+- Substrate Updates [C 604](https://github.com/darwinia-network/darwinia-common/pull/604)
+- Only For Darwinia Developers [C 19](https://github.com/darwinia-network/substrate-update-tracking/issues/19), [T 12](https://github.com/darwinia-network/substrate-update-tracking/issues/12), [T 15](https://github.com/darwinia-network/substrate-update-tracking/issues/15)
+#### Fix(es)
+- Prune `on-chain` MMR
+  - https://github.com/darwinia-network/darwinia-common/pull/673
+  - https://github.com/darwinia-network/darwinia-common/pull/689
+  - #675
```
Comment (AsceticBear) [inline CHANGELOG.md:45]: https://github.com/darwinia-network/darwinia-common/pull/675?

Comment (AurevoirXavier) [reply CHANGELOG.md:45]: Self repo link.

Comment (AurevoirXavier) [reply CHANGELOG.md:45]: #675

**Comment by AsceticBear** [comment]
> The new host function mentioned in polkadot v0.9.0? Any related pull request links to this break changes?

The test result show like this: https://github.com/darwinia-network/darwinia/pull/662#issuecomment-857621646

**Comment by HackFisher** [comment]
> > The new host function mentioned in polkadot v0.9.0? Any related pull request links to this break changes?
> 
> The test result show like this: [#662 (comment)](https://github.com/darwinia-network/darwinia/pull/662#issuecomment-857621646)

https://github.com/paritytech/substrate/pull/8655

**Comment by AurevoirXavier** [comment]
> > > The new host function mentioned in polkadot v0.9.0? Any related pull request links to this break changes?
> > 
> > 
> > The test result show like this: [#662 (comment)](https://github.com/darwinia-network/darwinia/pull/662#issuecomment-857621646)
> 
> [paritytech/substrate#8655](https://github.com/paritytech/substrate/pull/8655)

Already linked in CHANGELOG

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
+## [0.11.0] - 2021-07-20
+
+### IMPORTANT
+**Please upgrade your node as SOON as possible!! Otherwise, your node might not sync the new blocks after enacting runtime.**
+**请尽快升级您的节点！！否则您的节点将有可能不会同步新的区块。**
+
+#### Upgrade Window Period - 升级窗口期
+  - Crab UTC 2021-07-21 ~ 07-24-2021
```
Comment (AurevoirXavier) [inline CHANGELOG.md:8]: @HackFisher

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+  - Darwinia UTC 07-28-2021 ~ 08-01-2021
```
Comment (AurevoirXavier) [inline CHANGELOG.md:9]: @HackFisher

Comment (HackFisher) [reply CHANGELOG.md:9]: v0.11.0 darwinia在 mmr 裁剪期间的预期内存是多少?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,78 @@
+## [0.11.0] - 2021-07-20
+
+### IMPORTANT
+**Please upgrade your node as SOON as possible!! Otherwise, your node might not sync the new blocks after enacting runtime.**
+**请尽快升级您的节点！！否则升级后，您的节点将有可能不会同步新的区块。**
```
Comment (HackFisher) [inline CHANGELOG.md:5]:
```suggestion
**请尽快升级您的节点！！否则在Runtime升级后，您的节点将有可能不会同步新的区块。**
```

File: CHANGELOG.md (hunk 3 above)
Comment (AurevoirXavier) [reply CHANGELOG.md:9]: image

Avg: 0.8g ~ 1.2g

Recommend to reserve: 1.5g ~ 2.5g

File: CHANGELOG.md (hunk 4 above)
Comment (AsceticBear) [reply CHANGELOG.md:5]: ~在Runtime升级~ ->  在 Runtime 升级

Comment (AurevoirXavier) [reply CHANGELOG.md]: > ~在Runtime升级~ -> 在 Runtime 升级

Updated.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,78 @@
+## [0.11.0] - 2021-07-20
+
+### IMPORTANT
+**Please upgrade your node as SOON as possible!! Otherwise, your node might not sync the new blocks after enacting runtime.**
+**请尽快升级您的节点！！否则在 Runtime 升级后，您的节点将有可能不会同步新的区块。**
+
+#### Upgrade Window Period - 升级窗口期
+  - Crab UTC 07-21-2021 ~ 07-24-2021
```
Comment (HackFisher) [inline CHANGELOG.md:8]:
```suggestion
  - Crab Now ~ UTC 07-24-2021
```

File: CHANGELOG.md (hunk 3 above)
Comment (HackFisher) [reply CHANGELOG.md:9]:
```suggestion
  - Darwinia Now ~ UTC 07-28-2021
```

**Review by HackFisher — APPROVED** [review]

**Pull request merged**